grasp_apps/fixed_position_pick/launch/fixed_position_pick.yaml

```diff
@@ -0,0 +1,5 @@
+ur_test:
```

Comment:
can you check why depends on ur_test?

Reply:
Similar to fixed_position_pick. I missed to change the name.